proto/sentry_protos/snuba/v1/trace_item.proto

```diff
@@ -46,4 +46,5 @@ message TraceItem {
   // Internal fields
   uint32 retention_days = 100;
   google.protobuf.Timestamp received = 101;
+  uint32 downsampled_retention_days = 102;
```

Comment:
nit: can we add a comment on the difference between `retention_days` and `downsampled_retention_days`?